### ISSUE TITLE
Batch normalization arguments

### ISSUE THREAD

**Issue by TX2012LH** [vanhuyz/CycleGAN-TensorFlow#12]
Thank you for your wonderful code! Should the `updates_collections` argument in batch normalization should be set to `None` as told [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/layers/python/layers/layers.py)?

**Comment by TX2012LH** [comment]
Also when batch normalization is used, bias in every convolution layer can be ignored?

**Comment by vanhuyz** [comment]
You're right. `updates_collections` should be `None` and we can ignore biases in convolution layers. How about the case of instance normalization?  I think we can ignore biases too, what do you think?

**Comment by TX2012LH** [comment]
Yes, I think so.
